src/org/asia/hellokitty/Person.java

```diff
@@ -1,5 +1,6 @@
 package org.asia.hellokitty;
 
+// co robi ta klasa? WTF?
```

Comment:
Please translate this into English so that all can understand your comment.

Reply:
No.